.eslintrc.js

```diff
@@ -162,6 +162,8 @@ module.exports = {
 		// Allows Chai `expect` expressions. Now that we're on jest, hopefully we can remove this one.
 		'no-unused-expressions': 0,
 
+		'react/forbid-foreign-prop-types': 0,
```

Comment:
I think this is correct and available, but we should make sure addition of offending code creates a lint error. I didn't see any lint issues when I ran the linter locally.